account_statement_import_sheet_file/tests/test_account_statement_import_sheet_file.py

```diff
@@ -538,3 +538,51 @@ def test_amount_inverse_sign(self):
         line2 = statement.line_ids.filtered(lambda x: x.payment_ref == "LABEL 2")
         self.assertEqual(line2.amount, 1525.00)
         self.assertEqual(line2.amount_currency, 1000.00)
+
+    def test_import_xlsx_empty_values(self):
+        sample_statement_map_empty_values = (
+            self.AccountStatementImportSheetMapping.create(
+                {
+                    "name": "Sample Statement with empty values",
+                    "amount_type": "distinct_credit_debit",
+                    "float_decimal_sep": "comma",
+                    "delimiter": "n/a",
+                    "no_header": 0,
+                    "footer_lines_skip_count": 1,
+                    "amount_inverse_sign": 0,
+                    "header_lines_skip_count": 1,
+                    "quotechar": '"',
+                    "float_thousands_sep": "dot",
+                    "reference_column": "MOVIMIENTO",
+                    "description_column": "DESCRIP",
+                    "amount_credit_column": "DEBE",
+                    "amount_debit_column": "HABER",
+                    "balance_column": "SALDO",
+                    "timestamp_format": "%d/%m/%Y",
+                    "timestamp_column": "FECHAMOVI",
```

Comment:
not so important: but probably better to put the values of the previous 7 dict items in English.